File: backend/GymRecommender/models/city.py
from django.db import models
from .country import Country
import uuid

class City(models.Model):
    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
    nelatitude = models.FloatField()
    nelongitude = models.FloatField()
    swlatitude = models.FloatField()
    swlongitude = models.FloatField()
    name = models.CharField(max_length=100)
    country = models.ForeignKey(Country, models.DO_NOTHING)

    class Meta:
        managed = False
        db_table = 'city'

File: backend/Models/City.cs
using System;
using System.Collections.Generic;

namespace backend.Models;

public partial class City
{
    public Guid Id { get; set; }

    public double Nelatitude { get; set; }

    public double Nelongitude { get; set; }

    public double Swlatitude { get; set; }

    public double Swlongitude { get; set; }

    public string Name { get; set; } = null!;

    public Guid CountryId { get; set; }

    public virtual Country Country { get; set; } = null!;

    public virtual ICollection<Gym> Gyms { get; set; } = new List<Gym>();
}
